scripts/validate_aws_credentials.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 SCRIPT_DIR=$(cd "$(dirname "$0")" && pwd)
 
+if ${SKIP_AWS_CREDENTIAL_VALIDATION:-false} ; then
```

Comment:
This will actually execute the contents of the variable as a command. eg:

```
$ cat test.sh
#!/bin/bash

if ${FOO:-false} ; then
  echo "true"
  exit 0
fi

echo "false"

$ FOO=wibble ./test.sh 
./test.sh: line 3: wibble: command not found
false
```

It would be better to test for string values instead like we do elsewhere.